packages/react-core/src/components/DragDrop/DragDrop.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+interface ZoneNodes {
+  [zone: string]: HTMLElement[];
+}
+
+export const DragDropContext = React.createContext({
+  draggableZoneNodes: {} as ZoneNodes,
+  setDraggableZoneNodes: (_nodes: ZoneNodes) => {},
+  onDrop: (
+    _sourceDroppableKey: string | number,
+    _sourceDraggableKey: string | number,
+    _destDroppableKey: string | number | null,
+    _destDraggableKey: string | number | null
+  ) => false as boolean
+});
+
+interface DragDropProps {
+  /** Potentially Droppable and Draggable children */
+  children?: React.ReactNode;
```

Comment:
Are there any rules for the relationship between draggable and droppable zones? they each have their own keys - and the zone prop is used to enforce some sort of rule. I think I see how you implemented it in the basic examples, but I am having a hard time comprehending the limits or other ways I might use them.

Reply:
`Draggable`s go in `Droppable`s. `Droppable`s go in `DragDrop`s.

Reply:
Love it - can all that be explain in docs somehow?